frontend/src/CalendarWeekdayCell/CalendarWeekdayCell.tsx

```diff
@@ -0,0 +1,15 @@
+import classes from './CalendarWeekdayCell.module.css';
+
+type WeekdayCellProps = {
+    day?: string
+};
+
+function CalendarWeekdayCell(props: WeekdayCellProps) {
```

Comment:
the naming of this component is a little confusing; i think it implies that it represents a day which may have events in it rather than just displaying the name of a day. maybe i'm being fussy. i would change it to something like `CalendarDayName` or similar, personally